### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module : import axios from 'axios' 

### ISSUE THREAD

**Issue by serhatgemici** [zealous-tech/testrail-reporter#10]
Hi @gevorg-vardanyan-im,
I've followed the steps for installation and configuration on my playwright project and double checked it, however I hit this following issue when I execute my tests:
```
/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/@zealteam/testrail-reporter/src/testrailApi.js:1
import axios from 'axios';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:77:18)
    at wrapSafe (node:internal/modules/cjs/loader:1290:20)
    at Module._compile (node:internal/modules/cjs/loader:1342:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
    at Object.i.<computed>.ut._extensions.<computed> [as .js] (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/utilsBundleImpl.js:16:1010)
    at Module.load (node:internal/modules/cjs/loader:1212:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1028:12)
    at Module.require (node:internal/modules/cjs/loader:1237:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/@zealteam/testrail-reporter/src/base.js:2:16)
    at Module._compile (node:internal/modules/cjs/loader:1378:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
    at Object.i.<computed>.ut._extensions.<computed> [as .js] (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/utilsBundleImpl.js:16:1010)
    at Module.load (node:internal/modules/cjs/loader:1212:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1028:12)
    at Module.require (node:internal/modules/cjs/loader:1237:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/@zealteam/testrail-reporter/src/reporters/caller-vitest.js:4:19)
    at Module._compile (node:internal/modules/cjs/loader:1378:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
    at Object.i.<computed>.ut._extensions.<computed> [as .js] (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/utilsBundleImpl.js:16:1010)
    at Module.load (node:internal/modules/cjs/loader:1212:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1028:12)
    at Module.require (node:internal/modules/cjs/loader:1237:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/@zealteam/testrail-reporter/index.js:1:22)
    at Module._compile (node:internal/modules/cjs/loader:1378:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
    at Object.i.<computed>.ut._extensions.<computed> [as .js] (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/utilsBundleImpl.js:16:1010)
    at Module.load (node:internal/modules/cjs/loader:1212:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1028:12)
    at Module.require (node:internal/modules/cjs/loader:1237:19)
    at require (node:internal/modules/helpers:176:18)
    at requireOrImport (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/transform/transform.js:192:20)
    at requireOrImportDefaultFunction (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/runner/loadUtils.js:284:51)
    at loadReporter (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/runner/loadUtils.js:293:10)
    at createReporters (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/runner/reporters.js:67:69)
    at Runner.runAllTests (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/runner/runner.js:69:127)
    at runTests (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/program.js:207:98)
    at t.<anonymous> (/Users/serhat.gemici/Documents/PlaywrightProject/node_modules/playwright/lib/program.js:54:7)
```

Do you have a guess what might be the problem?

Thanks in advance & Best Regards

**Comment by arayik-chaparyan-im** [comment]
Thank you, @serhatgemici, for your interest in our module. We will try to reproduce the issue on our end and get back to you soon.

**Comment by NanoTechnolog3000** [comment]
Is there any news regarding this issue?
I faced the same error when trying to execute my tests:
`/Users/nanotechnolog/Documents/projects/playwrightProject/node_modules/@zealteam/testrail-reporter/src/testrailApi.js:1
import axios from 'axios';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:76:18)
    at wrapSafe (node:internal/modules/cjs/loader:1283:20)
    at Module._compile (node:internal/modules/cjs/loader:1328:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    ...
`

**Comment by arayik-chaparyan-im** [comment]
Hi @NanoTechnolog3000 Hi @serhatgemici 
Thank you for your interest in and use of our module. 

We have released a new version, 1.0.11, which fixes the identified error. Please try it out and let us know if you encounter any issues.

Feel free to share your suggestions and the features you would like to see in this module.

**Comment by arayik-chaparyan-im** [comment]
Closing the issue as it has been resolved and no further concerns have been raised.
